configs/keycloak_bridge.yml

```diff
@@ -196,6 +196,32 @@ recaptcha-url: https://www.google.com/recaptcha/api/siteverify
 recaptcha-secret: theverymysterioussecretfortherecaptchaverifyoperation
 sse-public-url: https://sse.trustid.ch
 
+# Corporate register
+corp-register-keys:
```

Comment:
As we did for validation, I think we should have a technical user in master realm which will be able to create user in any realm. It would avoid such kind of complex configuration. What do you think ?

Reply:
So I suggest to remove technical users defined for trustID and new endpoints and use a single one : technical-realm

Reply:
Yes, I think it is a good idea